heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraScheduler.java

```diff
@@ -184,9 +185,29 @@ public boolean onUpdate(Scheduler.UpdateTopologyRequest request) {
     return true;
   }
 
+
+  boolean confirmWithUser(int newContainerCount) {
```

Comment:
I'd prefer to use the name pattern like "hasConfirmedWithUser(int newContainerCount )", but it is up to you

Reply:
updated